crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -247,6 +257,7 @@ impl DatabaseEnv {
             // worsens it for random access (which is our access pattern outside of sync)
             no_rdahead: true,
             coalesce: true,
+            exclusive: inner_env.is_exclusive(),
```

Comment:
just passing it from the arguments should work

```suggestion
            exclusive: args.exclusive,
```